### ISSUE TITLE
Benchmarks won't open

### ISSUE THREAD

**Issue by xiaoyangyang123** [ScanNet/ScanNet#127]
Prompt the following sentence:
Hmm... can't access this page The page at https://kaldir.vc.in.tum.de/scannet_benchmark/submission_upload?id=1405 may have a problem or has moved permanently to a new URL.

**Comment by Fanyan-0411** [comment]
Hi, can you open the benchmark website now?

**Comment by xiaoyangyang123** [comment]
> Hi, can you open the benchmark website now?

No, yesterday the official email said, let’s see the notification on github

**Comment by Fanyan-0411** [comment]
Thanks a lot!

**Comment by xiaoyangyang123** [comment]
> Thanks a lot!

Excuse me, can you open the test site?

**Comment by Fanyan-0411** [comment]
> 

Yes, it's available.

**Comment by Fanyan-0411** [comment]
> 

There is something wrong with the submission procedure. 

After uploading the .zip or .7z file, we got 
 "Could not archive the uploaded file." 

I have checked the results file. Please let me know if you meet with the same problem. 

Thanks.

**Comment by xiaoyangyang123** [comment]
> > 
> 
> 提交程序有问题。
> 
> 上传.zip或.7z文件后，我们得到“无法存档上传的文件”。
> 
> 我已经检查了结果文件。如果您遇到同样的问题，请告诉我。
> 
> 谢谢。

I'm in the same situation, did you solve it?

**Comment by Fanyan-0411** [comment]
> 

Unfortunately not yet.

**Comment by fanyan121212** [comment]
Hi! Have you tested your model? Dose the testing takes several days?

**Comment by xiaoyangyang123** [comment]
> 嗨，您现在可以打开基准测试网站吗？

The results can be uploaded

**Comment by xiaoyangyang123** [comment]
> 你好！你测试过你的模型吗？剂量测试需要几天时间？

one day

**Comment by learner-zhou** [comment]
I want to test my 3D instance segmentation model for point clouds. I chose the Full Data Task Type: 3D semantic instance prediction and uploaded the file as the Format for 3D Semantic Instance Prediction. Why did it fail? Email shows 
"No text files found in results archive"

I have checked the results file. Please let me know if you meet with the same problem.
Thanks a lot!
